README.md

```diff
@@ -30,14 +30,21 @@ In this section you can find examples of how to use template workflows. For more
 
 The action is called by creating a PR. It is necessary that the repository is enabled for auto-merge.
 
+⚠️ If you want to enable the force merge, make sure that the app can bypass any protection rules.
```

Comment:
I would like to have more explanation on the difference between enabled and disabled "force". I think this is the most critical part, and we need to explain that `force: false` enables a GitHub feature that merges the PR when all conditions are met. With `force: true`, it will immediately merge the PR when the job is done.